v0/destinations/google_adwords_offline_conversions/data/TrackCallConversionsConfig.json

```diff
@@ -0,0 +1,41 @@
+[
+  {
+    "destKey": "conversions[0].callerId",
+    "sourceKeys": "properties.callerId",
+    "required": true
+  },
+  {
+    "destKey": "conversions[0].callStartDateTime",
+    "sourceKeys": "properties.callStartDateTime",
+    "required": true
```

Comment:
Can't we map it from `originalTimestamp` or `recievedAt`?

Reply:
We can keep those after this in terms of priority

Reply:
we are already mapping `conversionDateTime` with originalTimestamp.

Reply:
this can mix `conversion` will `call`

Reply:
Make sense